pkg/fleet-manager/plugin/plugin.go

```diff
@@ -213,6 +216,77 @@ func RendPrometheus(fsys fs.FS, fleetName types.NamespacedName, fleetRef *metav1
 	return renderFleetPlugin(fsys, promCfg)
 }
 
+type veleroObjectStoreLocation struct {
+	Bucket   string                          `json:"bucket"`
+	Provider string                          `json:"provider"`
+	Config   veleroObjectStoreLocationConfig `json:"config"`
+}
+type veleroObjectStoreLocationConfig struct {
+	S3Url            string `json:"s3Url"`
+	Region           string `json:"region"`
+	S3ForcePathStyle bool   `json:"s3ForcePathStyle"`
+}
+
+func RenderVelero(fsys fs.FS, fleetNN types.NamespacedName, fleetRef *metav1.OwnerReference, cluster FleetCluster, backupCfg *fleetv1a1.BackupConfig, accessKey, secretKey string) ([]byte, error) {
+	// get and merge the chart config
+	c, err := getFleetPluginChart(fsys, VeleroComponentName)
+	if err != nil {
+		return nil, err
+	}
+	mergeChartConfig(c, backupCfg.Chart)
+
+	// get default values
+	defaultValues := c.Values
+	if err != nil {
+		return nil, err
+	}
+
+	// get custom values
+	customValues := map[string]interface{}{
+		"configuration": map[string]interface{}{
+			"backupStorageLocation": []veleroObjectStoreLocation{
+				{
+					Bucket:   backupCfg.Storage.Location.Bucket,
+					Provider: backupCfg.Storage.Location.Provider,
+					Config: veleroObjectStoreLocationConfig{
+						S3Url:            backupCfg.Storage.Location.Endpoint,
+						Region:           backupCfg.Storage.Location.Region,
+						S3ForcePathStyle: true,
+					},
+				},
+			},
+		},
+		"credentials": map[string]interface{}{
+			"secretContents": map[string]interface{}{
+				"cloud": fmt.Sprintf(
+					"[default]\naws_access_key_id=%s\naws_secret_access_key=%s",
+					accessKey,
+					secretKey,
```

Comment:
We should not allow ak sk used like this, they should be referenced from a secret

Reply:
ok，we can just pass the secretName